bring-it-up/bring-it-up
# M1 search bar

## Description
#### Summary:
- Implements search bar and indexing 

#### Changes:
- Adds SearchBar.tsx file in component folder
- Adds SearchBar component and implements indexing logic in Home.tsx file

#### How to test this PR:

1. Go to our landing page 
2. The search bar + all of the service cards should display 
3. Type in a search string in the bar and press search button
4.  Service cards only relevant to the search string should be displayed 

## Notes
There is some weird behaviour with the search bar styling that needs to be addressed later down the road: 
- The borders of the search bar and button change when clicked
- The search bar + button placement is hardcoded and will need to be modified if it moves
## Screenshots:
<img width="726" alt="Screen Shot 2022-11-20 at 2 21 54 PM" src="https://user-images.githubusercontent.com/73853606/202929451-06630853-48c5-45a3-97ad-6e8046ceffc6.png">

## Checklist
